web/versioned_docs/version-0.13.0/auth/Pills.css

```diff
@@ -1,6 +1,7 @@
 :root {
```

Comment:
Please revert all the changes in the `0.13.0` versioned docs since this feature won't be included in the 0.13.0 version of Wasp.